openproblems/data/pancreas.py

```diff
@@ -39,10 +39,10 @@ def load_pancreas(test=False):
             scprep.io.download.download_url(URL, filepath)
             adata = sc.read(filepath)
 
-            # Remove preprocessing
+            # NOTE: X contains counts that are normalized with scran
+            adata.layers["log_scran"] = adata.X
             adata.X = adata.layers["counts"]
             del adata.layers["counts"]
```

Comment:
I would keep the `adata.layers['counts']` here.

Reply:
this somehow breaks the CI workflow if I leave it in

Reply:
CI should only test if `adata.layers['counts']` is the same as `adataX`. In the immune data loader, this is still the case though, no? So it should work here too, no? What is your error?

Reply:
The error happens in the label projection data loader, I'm getting and exit code 137 for this one which I think is an out of memory error. Not really sure why that happens

Reply:
```
E       AssertionError: docker exec f9626baac2ad /bin/bash /__w/SingleCellOpenProblems/SingleCellOpenProblems/test/docker_run.sh /__w/SingleCellOpenProblems/SingleCellOpenProblems/test/ /tmp/tmp6kqg7uqu/test_load_dataset.py
E       Return code 137
E       
E       DEBUG:openproblems:Loading pancreas_batch dataset
E       DEBUG:openproblems:Downloading load_pancreas((True,), {}) dataset
E       DEBUG:openproblems:Downloading load_pancreas((False,), {}) dataset
E       /__w/SingleCellOpenProblems/SingleCellOpenProblems/test/docker_run.sh: line 18:   100 Killed                  python3 -m
```